recipe/meta.yaml

```
@@ -93,7 +94,10 @@ requirements:
     - decorator
     - prometheus_client
     # Required at runtime for their dynamically shared objects.
-    - folly
+    # Pin to prevent errors such as:
+    # E   ImportError: .../arcticdb_ext.cpython-39-x86_64-linux-gnu.so: undefined symbol: 
+    # _ZNK5folly7futures6detail8CoreBase11getExecutorEv
+    - folly ==2023.08.07.00
```

Variants using `libevent=2.1.10` fails because of constraints on dependencies. Previously they were using `folly=2023.03.20`. Let's try this version?
```suggestion
    - folly ==2023.08.07.00
```